docs/document-level-security/NETWORK_DRIVE.md

```diff
@@ -0,0 +1,27 @@
+### Setting up the Network Drive connector
+
+See the [Developer guide](../../docs/DEVELOPING.md) for setting up connectors.
+
+## Document level security
+
+Document Level Security (DLS) enables you to restrict access to documents based on a user's permissions.
+This feature is available by default for the Network Drive connector.
+Network drive DLS supports folder & file permissions that are shared with individuals or groups.
```

Comment:
This sentence could be a little clearer. 

We also to be consistent with capitalization "Network Drive connector", whereas use "network drive" when referring to concrete instances (for example "Windows network drive", "learn how to search data on your network drive").